sdk/ai/ai-inference-rest/generated/modelClient.ts

```diff
@@ -44,7 +44,6 @@ export default function createClient(
     },
     credentials: {
       scopes: options.credentials?.scopes ?? ["https://ml.azure.com/.default"],
-      apiKeyHeaderName: options.credentials?.apiKeyHeaderName ?? "api-key",
```

Comment:
as long as we make sure a merge doesn't overwrite this in src, it's fine

Reply:
Yeah, and I think the v2 customization script will help with that, it should call out conflicts between the autogen and custom code.